coordinator/core/marbleapi_test.go

```diff
@@ -295,6 +237,22 @@ func (ms *marbleSpawner) newMarbleAsync(marbleType string, infraName string, sho
 	}()
 }
 
+func (ms *marbleSpawner) verifyCertificateFromEnvironment(envName string, params *rpc.Parameters, opts x509.VerifyOptions) x509.Certificate {
```

Comment:
Shoud this return an error, panic or is the current way (assert inside the function) okay?

Reply:
okay, but see comment regarding `require` below